concent_api/api-integration-force-get-task-result-test.py

```diff
@@ -1,132 +1,147 @@
 #!/usr/bin/env python3
-
+import argparse
 import os
 import sys
 import datetime
 import hashlib
-import time
 import random
-from base64                 import b64encode
-
-from golem_messages         import message
-from golem_messages         import shortcuts
+import requests
+import time
 
-from utils.helpers          import get_current_utc_timestamp
-from utils.testing_helpers  import generate_ecc_key_pair
+from base64                          import b64encode
+from golem_messages                  import message
+from golem_messages                  import shortcuts
+from golem_messages.message.concents import AckForceGetTaskResult, ForceGetTaskResultUpload, ForceGetTaskResultFailed
 
-from api_testing_helpers    import api_request
-from api_testing_helpers    import timestamp_to_isoformat
+from utils.helpers                   import get_current_utc_timestamp
+from utils.testing_helpers           import generate_ecc_key_pair
 
-from freezegun              import freeze_time
-import requests
+from api_testing_common              import api_request, parse_command_line, create_task_to_compute, get_protocol_constants, \
+    print_protocol_constants
 
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "concent_api.settings")
 
+DEADLINE_OFFSET        = 5
+CONCENT_MESSAGING_TIME = 30
+FORCE_ACCEPTANCE_TIME  = 30
+TOKEN_EXPIRATION_TIME  = 3600
+
+WAIT_TIME_1            = DEADLINE_OFFSET + FORCE_ACCEPTANCE_TIME
+
 (PROVIDER_PRIVATE_KEY,  PROVIDER_PUBLIC_KEY)  = generate_ecc_key_pair()
 (REQUESTOR_PRIVATE_KEY, REQUESTOR_PUBLIC_KEY) = generate_ecc_key_pair()
 
 
-def parse_command_line(command_line):
-    if len(command_line) <= 1:
-        sys.exit('Not enough arguments')
+def prepare_hash(file_check_sum, hash_offset):
+    return file_check_sum[:-1] + hex(int(file_check_sum[-1], 16) + hash_offset)[2:]
```

Comment:
Please add a comment what this function does, name is good, but if this is some common method used to prepare hash it is worth describing.

Reply:
done